client/Dockerfile

```diff
@@ -2,14 +2,31 @@ FROM node:11.5-alpine
 
 RUN mkdir -p /usr/src/client
 
+ARG PUPPETEER_VERSION=1.10.0
+
 WORKDIR /usr/src/client
 
-RUN yarn global add @api-platform/client-generator
+RUN apk add --no-cache ca-certificates python bash make g++ chromium; \
```

Comment:
Are python, make and g++ required after build time? If it's only to build the GYP modules, you can use this trick: https://github.com/api-platform/api-platform/blob/master/admin/Dockerfile#L10-L15

Reply:
No, they are required by puppeteer & chromium on runtime